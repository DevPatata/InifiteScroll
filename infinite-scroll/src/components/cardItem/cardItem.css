.card {
  display: grid;
  grid-template-columns: 1fr 1fr;
  align-items: center;
  border-radius: 5px;
  background-color: #fff;
  box-shadow: 0 2px 5px 0 rgba(0, 0, 0, 0.16), 0 2px 10px 0 rgba(0, 0, 0, 0.12);

  &:hover {
    box-shadow: 0 8px 17px 0 rgba(0, 0, 0, 0.2), 0 6px 20px 0 rgba(0, 0, 0, 0.19);
  }
}

img {
  border-radius: 5px;
}