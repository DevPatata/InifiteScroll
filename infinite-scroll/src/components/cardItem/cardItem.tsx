import React from 'react'
import './cardItem.css'

export function CardItems({ children }) {

  return (
    <div className="card">
      {children}
    </div>
  )
}
