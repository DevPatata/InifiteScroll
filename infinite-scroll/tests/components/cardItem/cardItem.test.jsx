import React from 'react'
import { render, screen } from '@testing-library/react';
import { CardItems } from '../../../src/components/cardItem/cardItem';


describe('Prueba en componente CardItem', () => {

  test('Prueba match snaptshot', () => {
    const { container } = render(<CardItems />);

    expect(container).toMatchSnapshot();  
  })

  test('Prueba debe de contener un texto "TestCardItem"', () => {
    const children = 'TestCardItem';

    render( <CardItems children={children}/>);
    
    expect( screen.getByText('TestCardItem') ).toBeTruthy();
  })
})