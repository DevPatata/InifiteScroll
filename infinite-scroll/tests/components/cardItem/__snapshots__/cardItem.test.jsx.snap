// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Prueba en componente CardItem Prueba match snaptshot 1`] = `
<div>
  <div
    class="card"
  />
</div>
`;
